jaeger-core/src/main/java/com/uber/jaeger/Constants.java

```diff
@@ -42,4 +42,19 @@
    * root span, so that the trace can be found in the UI using this value as a correlation ID.
    */
   public static final String DEBUG_ID_HEADER_KEY = "jaeger-debug-id";
+
+  /**
+   * The name of the tag used to report client version.
+   */
+  public static final String JAEGER_CLIENT_VERSION_TAG_KEY = "jaeger.version";
+
+  /**
+   * The name used to report host name of the process.
+   */
+  public static final String TRACER_HOSTNAME_TAG_KEY = "jaeger.hostname";
```

Comment:
I think this should be just `hostname`. It needed to be `jaeger.xxx` when we were sending Zipkin format since it did not have the distinction between span and process tags.